### ISSUE TITLE
Connect to Github Hangs

### ISSUE THREAD

**Issue by jonathanforhan** [git-ecosystem/git-credential-manager#1013]
### Version

2.0.891+233ac2949b

### Operating system

Linux

### OS version or distribution

Fedora Workstation 37

### Git hosting provider(s)

GitHub

### (Azure DevOps only) What format is your remote URL?

None

### Can you access the remote repository directly in the browser?

Yes, I can access the repository

### Expected behavior

I expected to login in to push to master and login like normal

### Actual behavior

I pushed to master and GCM popped up and hung I had to cancel from cli

[](url)
![Screenshot from 2023-01-02 22-53-30](https://user-images.githubusercontent.com/99012095/210298706-7ee7f854-30ec-41c2-b0e6-6d129599a62f.png)

### Logs

Running diagnostics...

 [ OK ] Environment
 [ OK ] File system
 [ OK ] Networking
 [ OK ] Git
 [ OK ] Credential storage
 [ OK ] Microsoft authentication (AAD/MSA)
 [ OK ] GitHub API

Diagnostic summary: 7 passed, 0 skipped, 0 failed.
Log files:
  /home/jon/dev/csock/gcm-diagnose.log

Caution: Log files may include sensitive information - redact before sharing.

**Comment by mjcheetham** [comment]
Hello @jonathanforhan!

Thanks for opening this issue. This looks to be a problem with Avalonia UI.

> 2.0.891+233ac2949b

I don't recognise this GCM version number.. are you using a fork or custom build? Did you acquire GCM via the install from source script or some other means?

Please can you try and run the following command to open a UI test prompt, and see if this reproduces (assuming GCM was installed to `/usr/local/share/gcm-core`)?

```shell
/usr/local/share/gcm-core/GitHub.UI
```

You should see something like this (screenshot taken on a Mac, but should appear the same on Linux distributions):

![image](https://user-images.githubusercontent.com/5658207/210409966-37d4fb16-1c85-4e57-b58e-44e302a2632c.png)

Clicking the 'Show' button on the 'Credentials' tab should show this window:

![image](https://user-images.githubusercontent.com/5658207/210410022-ec38c48c-3b9a-457c-92dd-20385cde7dea.png)

**Comment by jonathanforhan** [comment]
Hello thank you for response

this is from the install link on this repo's README. Originally I installed from source (using the install-from-source.sh), that was the screenshot that I included. I then also installed via the amd64 tarball download that is also provided, same issue. however my version now says '2.0.886+ea93cb5158', sorry i didn't realize that there would be a difference.

when run the script for test UI is is same story, window opens black and hangs:

![Screenshot from 2023-01-03 15-14-24](https://user-images.githubusercontent.com/99012095/210434879-32691209-74b2-4869-9d25-f148ba6ac9f1.png)

my current workaround is to open project in vscode and authenticate that way which allows me to push without issue after I ran 'git credential-manager unconfigure'
